insights/parsers/pmrep.py

```diff
@@ -0,0 +1,80 @@
+"""
+Pmrep - command ``pmrep -t 1s -T 1s network.interface.out.packets network.interface.collisions swap.pagesout``
+==============================================================================================================
+
+Parse the content of the ``pmrep -t 1s -T 1s network.interface.out.packets network.interface.collisions swap.pagesout`` command.
+
+Sample ``pmrep -t 1s -T 1s network.interface.out.packets network.interface.collisions swap.pagesout`` command output::
+
+    n.i.o.packets  n.i.o.packets  n.i.collisions  n.i.collisions  s.pagesout
+             eth0             lo            eth0              lo
+          count/s        count/s         count/s         count/s     count/s
+              N/A            N/A             N/A             N/A         N/A
+            1.997          0.000           0.000           0.000       0.000
+
+Examples:
+    >>> type(pmrep_doc_obj)
+    <class 'insights.parsers.pmrep.PMREPMetrics'>
+    >>> pmrep_doc_obj.data.get('n.i.o.packets', None)
+    [{'eth0': '1.000'}, {'lo': '2.000'}]
+    >>> pmrep_doc_obj.data.get('n.i.collisions', None)
+    [{'eth0': '3.000'}, {'lo': '4.000'}]
+    >>> pmrep_doc_obj.data.get('s.pagesout', None)
+    ['5.000']
+"""
+
+from insights import Parser, parser
+from insights.specs import Specs
+from insights.parsers import SkipException
+
+
+@parser(Specs.pmrep_metrics)
+class PMREPMetrics(Parser):
+    """Parses output of ``pmrep -t 1s -T 1s network.interface.out.packets network.interface.collisions swap.pagesout`` command."""
+    def parse_content(self, content):
+        if not content:
+            raise SkipException("Empty content.")
+
+        heading_keys = ["n.i.o.packets", "n.i.collisions", "s.pagesout"]
+        ignore_values = ["count/s", "N/A"]
+        header_value = []
+        self.data = {}
```

Comment:
Since the `self.data` is not a clear attribute to indicate the data type of the parser, it's not recommended anymore. Please make this parser a `list` of `dicts` by inheriting from `list` directly.

For ex:
- https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/satellite_postgresql_query.py

Reply:
Ack, I will make this parser as a list of dicts.